### PR TITLE
Add barre separate and fix styling

### PR DESCRIPTION
**Pull request by francksefu**
Add any missing JSX elements to your project (without adding extra functionality) -> done
Your final design should match Zeplin's design (this includes all elements + the same styling) -> done
You may use the CSS rules detailed in Zeplin to style the website. -> done

### CODE REVIEW

File: src/components/BooksRender.jsx [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+        <div>
+          <span><h2 className="titre">Bookstore CMS</h2></span>
+          <span className="books">BOOKS</span>
+          <span className="categorie">CATEGORIES</span>
+        </div>
+        <div>
+          <img src={imagete} alt="person" className="Oval-tete" />
+        </div>
+      </div>
+      <BooksList />
```
Comment (Tufahel) [inline src/components/BooksRender.jsx:18]: - [ ] Inside `BooksList` component you have used `ul` and `li` which is correct. But by default `ul` and `li` always produce a `dot` before any `item`. I would recommend you use `no-style` CSS class to remove the `dot`. Kindly fix this issue. Consider going through this [content](https://gomakethings.com/creating-unstyled-lists/) for help.

- image

File: src/App.css [hunk 2]
```diff
@@ -1,7 +1,76 @@
-h2.range {
-  text-align: center;
-  margin-top: 20px;
-  margin-bottom: 20px;
+:root {
+  --white-two: #fff;
+  --azure: #0290ff;
+}
+body {
+  width: 100rem;
```
Comment (Tufahel) [inline src/App.css:11]: - [ ] I need to scroll from `left to right` to see the whole content. That means the whole content is not `fitting` within my browser interface. That shouldn't be the issue as per the given design. Kindly fix this issue.

image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @francksefu  :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Follow the correct Git flow.
✔️ Application is working fine.

## Required Changes ♻️

- [ ] I can see the name of the project is given wrong in the `Readme`. It is not the leaderboard project. Kindly fix this issue to make it a professional readme.

image


***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

- [ ] ***[OPTIONAL]***  I can see this is not your first pull request. It is recommended and must to do the same task within the same `PR` and not merge or close before the approval. If by mistake it is closed or merged before approval then we need to add the previous `PR` link with the new `PR`. Kindly remember it for future.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/BooksRender.jsx (hunk 1 above)
Comment (alexander16108) [inline src/components/BooksRender.jsx:18]: good job on removing the scroll on the webpage but that doesn't solve the problem of the whole content not fitting within my browser interface. please fix this and try again 

image

**Review by alexander16108 — CHANGES_REQUESTED** [review]
Hi @francksefu , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
✔️ Descriptive pull request.
✔️ Follow the correct Git flow.
✔️ Application is working fine.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @francksefu, 👋 👋 

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous revieer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




### To Highlight :1st_place_medal: 

- Descriptive Pull Request title and summary. ✔️
- Styling of the bookstore CMS project ✔️
- Zeplin Design template used for the design ✔️
- There is good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**